pom.xml

```diff
@@ -64,11 +64,12 @@
     <properties>
         <revision>3.8</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.222.4</jenkins.version>
+        <jenkins.version>2.273-rc30689.09147672b85b</jenkins.version>
```

Comment:
Can you please link to whatever Jenkins core PR produced this incremental so reviewers have more context about the proposed change? Thanks!